data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,391 @@ mission "Remnant: Face to Maw 2B"
+			"Forge-Class Steering"
+			"Crucible-Class Steering"
+			"Hyperdrive"
+
+		engine -16 82 0.7
+		engine 16 82 0.7
+		engine 0 90 1
+		gun 0 -99 "Inhibitor Cannon"
+		gun -12 -72 "Inhibitor Cannon"
+		gun 12 -72 "Inhibitor Cannon"
+		gun -18 -49
+		gun 18 -49
+		turret 0 2 "Point Defense Turret"
+		leak "remnant leak" 30 4
+		leak "remnant leak sparkle" 30 10
+		explode "tiny explosion" 40
+		explode "small explosion" 20
+		explode "medium explosion" 10
+		"final explode" "final explosion small" 1
+		description "One of the strangest and most useful things that the Remnant discovered in the abandoned worlds of the Ember Waste is alien cloaking technology. They designed their scout ships to be able to cloak when needed, in order to explore alien territory without being seen... or to escape in a hurry after stealing something particularly valuable from those aliens."
+		description "	The cloaking technology is built into the Starling's hull; it cannot be removed or transferred to another ship."
+
+
+
+mission "Remnant: Crystal Research 5"
+	name "Scan the <npc> in <waypoints>"
+	description `Go scan the <npc> to record the Keystone post-wormhole.`
+	landing
+	waypoint "Peragenor"
+	source "Viminal"
+	destination "Viminal"
+	npc "scan outfits" "save"
+		government "Remnant"
+		personality coward disables plunders staying surveillance target mining
+		system "Peragenor"
+		ship "Starling (Keystone)" "Winds of Light"
+	to offer
+		has "Remnant: Crystal Research 4: done"
+	on offer
+		conversation
+			`You exit the ship to find that the pilot is actually Aeolus, and he has already marshalled a few dock workers with a number of wagons and lifters to unload the cargo.`
+			`	"Ah, <first>, thank you for retrieving all these Keystones." he pauses to stare at a crate bearing the image of a bunch of smiling Hai, then shakes his head and turns back to you.`
+			`	"Our researchers have decided that we need to test how a ship performs with one of these Keystones compared to a regular Key Stone. My ship has been outfitted with a keystone built from the samples you brought us earlier. We request that you meet us in Peragenor, where you will scan us after going through the wormhole. Then we will return here, swap out the Quantum Keystone built with Hai crystals for a Quantum Key Stone made from our crystals. Then we go back to Peragenor and get scanned again. By using the same ships for both tests, we should be able to get the best data."`
+			choice
+				`	"Wait, you are a pilot?"`
+				`	"OK"`
+					accept
+			`	"Oh, yes. It is actually quite a common skill among the Remnant. I am not a particularly good pilot, but it is sufficiently important that it is taught to almost everyone as a general skill. Especially those of us who work directly with ships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures carry tones of both pride and depreciation as he talks about his piloting skills. "Anyway, let's go"`
```

Comment:
`sufficiently` could he removed here. It took me a second to realize what was being explained because of that word usage. Just `important` might be better, but again, it's subjective. The second sentence (starts with `Especially`) is a fragment. Might I suggest: `It's especially worthwhile for those of us who work directly with ships on a daily basis and could be called upon to move or fly at a moment's notice.` Is `depreciation` a typo, or intentional? If intentional, you may want to consider a different word, as depreciation is more often associated with diminishing value of a certain item over time than it is the opposite of appreciation.

Reply:
I changed the third sentence; and swapped out the "depreciation" for "humility".

New version:
> I am not a particularly good pilot, but it is considered an important skill that is taught to almost everyone as a general part of general education. It is especially worthwhile for those of us who work directly with ships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures carry tones of both pride and humility as he talks about his piloting skills. "Anyway, let's go"

Reply:
Looks good! Just forgot a period at the end there.